packages/studio/server/src/studio/utils/bpfs.ts

```diff
@@ -93,5 +96,38 @@ export const Instance: bpfs = {
   },
   moveFile(fromPath: string, toPath: string): Promise<void> {
     return fse.move(resolvePath(fromPath), resolvePath(toPath))
+  },
+  async exportToArchiveBuffer(): Promise<Buffer> {
```

Comment:
**nitpick**: although writing on disk is totally fine on the studio app we could have done this in all in mem. (not to be fixed)